test/integration/scheduler/queueing/queue.go

```diff
@@ -2065,6 +2070,32 @@ var CoreResourceEnqueueTestCases = []*CoreResourceEnqueueTestCase{
 	{
 		Name:         "Pod rejected with PVC by the CSI plugin is requeued when the related PVC is added",
 		InitialNodes: []*v1.Node{st.MakeNode().Name("fake-node").Label("node", "fake-node").Obj()},
+		OverridingSchedulerProfile: &configv1.KubeSchedulerProfile{
+			SchedulerName: ptr.To("default-scheduler"),
+			Plugins: &configv1.Plugins{
+				MultiPoint: configv1.PluginSet{
+					Enabled: []configv1.Plugin{
+						{Name: "VolumeBinding"},
+						// These are required plugins to start the scheduler.
+						{Name: "PrioritySort"},
+						{Name: "DefaultBinder"},
+					},
+					Disabled: []configv1.Plugin{
+						{Name: "*"},
+					},
+				},
+			},
+			PluginConfig: []configv1.PluginConfig{
```

Comment:
So, do we have to pass the config as well? What if we use the default config for the volume binding plugin?

Reply:
As I recall, it would have been impossible to pass with the empty config for VolumeBinding, but it was reviewed and passed 🤷